front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -73,20 +74,21 @@ const StdcmOperationalPoint = ({ point, opPointId, disabled }: StdcmOperationalP
           },
           [] as { label: string; id: string }[]
         ),
-    [point, sortedSearchResults]
+    [location, sortedSearchResults]
   );
 
   const dispatchNewPoint = (p?: SearchResultItemOperationalPoint) => {
```

Comment:
I know it doesn't come from you but would you mind ? :)
```suggestion
  const dispatchNewPoint = (searchedOp?: SearchResultItemOperationalPoint) => {
```

Reply:
I plan to do a PR to refacto a bit this component, can this wait until this PR ? :)